src/components/searchBar.vue

```diff
@@ -2,32 +2,59 @@
   <div class="search-wrapper">
```

Comment:
Use or deprecate?

Reply:
This is needed it's just the overarching div

Reply:
Oh as in the entire component, but looks like its needed for event searching.